qiskit_nature/circuit/library/ansatzes/succd.py

```diff
@@ -179,3 +221,53 @@ def _validate_num_particles(self, num_particles):
                 "differing numbers of alpha and beta electrons:",
                 str(num_particles),
             ) from exc
+
+    def _build_fermionic_excitation_ops(self, excitations: Sequence) -> List[FermionicOp]:
+        """Builds all possible excitation operators with the given number of excitations for the
+        specified number of particles distributed in the number of orbitals.
+        Args:
+            excitations: the list of excitations.
+        Returns:
+            The list of excitation operators in the second quantized formalism.
+        """
+        operators = []
+        excitations_dictionary: Dict[str, List[Tuple[Tuple[int, ...], Tuple[int, ...]]]] = {}
+        # Reform the excitations list to a dictionary. Each items in the dictionary
+        # corresponds to a parameter.
+        for exc in excitations:
+            exc_level = (
+                str(exc[0][0])
+                + str(exc[0][1])
+                + str(abs((exc[1][0] - exc[0][0]) - (exc[1][-1] - exc[0][-1])))
+                + str(exc[1][0] - exc[0][0] + exc[1][-1] - exc[0][-1])
+            )
+            # exc_level is a 4-number string. First two numbers are occupied indices. And
+            # last two numbers indicate the indices changing during excitation. Thus,
+            # the level of an excitations is indicated by this string. The symmetrically
+            # mirrored double excitations have the same exc_level string, and the
+            # excitations with the same level will be assigned the same parameter.
```

Comment:
This is an interesting solution. However, some updates will be required:
- this will break when `include_singles=True`. So you will need to guard against that (ideally add a unittest for this)
- could you maybe rephrase this comment in form of an equation that is a bit easier to read?